docker-compose.yaml

```diff
@@ -373,9 +509,11 @@ volumes:
   kibana:
   operate_tmp:
   tasklist_tmp:
+  postgres-web:
 
 networks:
-  # Note there are two bridge networks: One for Camunda Platform and one for Identity.
-  # Identity and Keycloak are part of both as they need to be accessible by platform components.
+  # Note there are three bridge networks: One for Camunda Platform, one for Identity and one for Web Modeler.
+  # Identity and Keycloak are part of the first two as they need to be accessible by platform components.
   camunda-platform:
   identity-network:
+  modeler:
```

Comment:
💭 Do we want to call this `web-modeler` instead?

Reply:
Should we rename every "modeler" occurrence to "web-modeler" too, then? For example, the service name `modeler-websocket` -> `web-modeler-websockets`?

Reply:
Yes, let's do that 👍